api/uniter/uniter.go

```diff
@@ -395,6 +395,51 @@ func (st *State) AllMachinePorts(machineTag names.MachineTag) (map[network.PortR
 	return portsMap, nil
 }
 
+// OpenedMachinePortRanges returns all port ranges currently open on the given
+// machine, grouped by unit tag and application endpoint.
+// relation that applies.
```

Comment:
Partial sentence.  Does it still belong?

Reply:
Whoops :-(